src/_locales/en/messages.json

```diff
@@ -257,7 +257,7 @@
     "welcome_1a4_type_auto_subtitle2": "This allows your friends to verify the connection between your social network account and your keypair.",
     "welcome_1a4_type_auto_switch": "Prefer doing it manually?",
     "welcome_1a4_type_manual_goto": "Copy & Go to Profile",
-    "welcome_1a4_type_manual_subtitle1": "Add this to bio, or post on timeline, before your first encrypted post.",
+    "welcome_1a4_type_manual_subtitle1": "Add this to your bio, or post on your timeline, before your first encrypted post.",
```

Comment:
This is related to proof post? If so, now it should be "Add this to your bio before your first encrypted post.".